config/analytics_custom_events.yml

```diff
@@ -18,6 +19,9 @@ shared:
   - jobseeker_application_submitted
   - jobseeker_subscription_update
   - jobseeker_confirmation_instructions
+  - jobseeker_inactive_account
+  - jobseeker_account_closed
+  - jobseeker_job_listing_ended_early
```

Comment:
Question: :raising_hand_man: 
Are these new events we want to send to send? 
Just trying to figure out if this is strictly related to the email events or just using the PR to add some extra info on top of triggering email events.